src/commands/slash/anony.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  Client,
+  CommandInteraction,
+  EmbedBuilder,
+} from "discord.js";
+import ANONY_CONSTANTS from "../../constants/anony";
+import { SlashCommand } from "../../types/utils";
+
+export default {
+  name: ANONY_CONSTANTS.name,
+  description: ANONY_CONSTANTS.description,
+  type: "command",
+  builder: (command) =>
+    command.addStringOption((option) =>
+      option
+        .setName(ANONY_CONSTANTS.subCommandTitleName)
+        .setDescription(ANONY_CONSTANTS.subCommandTitleDescription)
+        .setRequired(true),
+    )
+      .addStringOption((option) =>
+        option
+          .setName(ANONY_CONSTANTS.subCommandMessageName)
+          .setDescription(ANONY_CONSTANTS.subCommandMessageDescription)
+          .setRequired(true),
+      ),
+  async execute(client: Client, interaction: CommandInteraction) {
+    const channel = client.channels.cache.get(
+      process.env.ANONYMOUS_APPROVAL_CHANNEL_ID,
+    );
+    if (channel?.isTextBased()) {
+      const embed = new EmbedBuilder()
+        .setColor(ANONY_CONSTANTS.defaultColorCode)
+        .setTitle(
+          `(#${new Date().getTime()}) ${
+            interaction.options.data.find(
+              (data) => data.name === ANONY_CONSTANTS.subCommandTitleName,
+            )?.value ?? "Untitled"
+          }`,
+        )
```

Comment:
May I know the reason using a timestamp as prefix? Looks unnecessary to me.
We can remove it or leave the formatted datetime at the below of the content.
```suggestion
        .setTitle(
            interaction.options.data.find(
              (data) => data.name === ANONY_CONSTANTS.subCommandTitleName,
            )?.value ?? "Untitled"
        )
```

Reply:
make sense, but I will prefer to use an auto-increment id. Anyway, there is no need to add the id in this PR in my opinion(since it is big enough) let's launch it first and we can do more update in future.

Reply:
But how would you do auto-increment id without a DB? If you say fetch by current post ID, there is a possible concurrency issue. Which risk is increasing by the amount of members

Reply:
This could be an issue, so i prefer discuss it in separate issue and solve this in future. Currently, they can use post link or message link to mention others' post, what do you think?

Reply:
You got a point